doc/source/api.rst

```diff
@@ -564,21 +556,23 @@ following usable methods and properties (all available as ``Series.cat.<method_o
 .. autosummary::
    :toctree: generated/
 
-   Categorical.categories
-   Categorical.ordered
-   Categorical.rename_categories
-   Categorical.reorder_categories
-   Categorical.add_categories
-   Categorical.remove_categories
-   Categorical.remove_unused_categories
-   Categorical.set_categories
-   Categorical.codes
+   Series.cat.categories
+   Series.cat.ordered
+   Series.cat.rename_categories
+   Series.cat.reorder_categories
+   Series.cat.add_categories
+   Series.cat.remove_categories
+   Series.cat.remove_unused_categories
+   Series.cat.set_categories
+   Series.cat.codes
 
 To create a Series of dtype ``category``, use ``cat = s.astype("category")``.
 
```

Comment:
maybe move this to the Series section and/or these the original here - these are valid Crategorical properties that should be listed